collectors/python.d.plugin/anomalies/anomalies.chart.py

```diff
@@ -0,0 +1,308 @@
+# -*- coding: utf-8 -*-
+# Description: anomalies netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import time
+from datetime import datetime
+import re
+
+import requests
+import numpy as np
+import pandas as pd
+from netdata_pandas.data import get_data, get_allmetrics
+from pyod.models.hbos import HBOS
+from pyod.models.pca import PCA
+from pyod.models.loda import LODA
+from pyod.models.iforest import IForest
+from pyod.models.cblof import CBLOF
+from pyod.models.feature_bagging import FeatureBagging
+from pyod.models.copod import COPOD
+from sklearn.preprocessing import MinMaxScaler
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
+
+priority = 50
```

Comment:
I think it's a wrong priority. Even the header of system gages is broken. The priority should be much lower.

Reply:
maybe something like `200100` so is just below "Netdata Monitoring" ?

Reply:
I don't think it should be that low. It's ok to have it somewhere among other python modules.

Reply:
just below the System section is another obvious place - but i don't really think i should mess with the order of the menus or anything. dont want to break any users flow of System, CPUs, Memory, Disk, etc. I like below netdata monitoring as its somewhat related to that and then you can just scroll down to it in typical set up. 

I've no real strong preference on it though given user can easily change it if they want.